packages/snaps-cli/src/webpack/utils.test.ts

```diff
@@ -88,3 +91,33 @@ describe('pluralize', () => {
     expect(pluralize(2, 'test', 'problems')).toBe('problems');
   });
 });
+
+describe('getFallbacks', () => {
```

Comment:
Can you add a test to ensure that Node.js builtins that we don't polyfill are added to the list?